ports/esp32/boards/UM_TINYWATCHS3/sdkconfig.board

```diff
@@ -0,0 +1,19 @@
+CONFIG_ESPTOOLPY_FLASHMODE_QIO=y
+CONFIG_ESPTOOLPY_FLASHFREQ_80M=y
+CONFIG_ESPTOOLPY_AFTER_NORESET=y
+
+CONFIG_ESPTOOLPY_FLASHSIZE_4MB=
+CONFIG_ESPTOOLPY_FLASHSIZE_8MB=y
+CONFIG_ESPTOOLPY_FLASHSIZE_16MB=
+CONFIG_SPIRAM_MEMTEST=
+CONFIG_PARTITION_TABLE_CUSTOM=y
+CONFIG_PARTITION_TABLE_CUSTOM_FILENAME="partitions-8MiB.csv"
+
+CONFIG_LWIP_LOCAL_HOSTNAME="UMTinyWATCHS3"
+
+CONFIG_TINYUSB_DESC_CUSTOM_VID=0x303A
```

Comment:
To use a custom VID/PID you need to add two extra options as per this commit: 1cd61149e4328154110d4b1e38bcbecdcc0ec386

Reply:
Interesting, I had my S3 boards setup the same as I PR'd here and it worked as expected, but I added the extra lines for all of the boards that use a custom PID.

Thanks for the pointer.